### PR TITLE
Flutter Device Mode AppCenter

### PR DESCRIPTION
**Pull request by desusai7**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: integrations/rudder-integration-appcenter-flutter/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# appcenter
```
Comment (arnab-p) [inline integrations/rudder-integration-appcenter-flutter/README.md:1]:
```suggestion
# AppCenter
```

File: integrations/rudder-integration-appcenter-flutter/android/build.gradle [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    defaultConfig {
+        minSdkVersion 21
+    }
+}
+
+dependencies {
+    // rudderstack dependencies
+    implementation 'com.rudderstack.android.sdk:core:1.+'
+    implementation 'com.rudderstack.android.integration:appcenter:1.0.1'
```
Comment (arnab-p) [inline integrations/rudder-integration-appcenter-flutter/android/build.gradle:35]:
```suggestion
    implementation 'com.rudderstack.android.integration:appcenter:1.+'
```

File: integrations/rudder-integration-appcenter-flutter/example/android/app/src/main/java/com/rudderstack/sdk/flutter/integrations/appcenter_example/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package com.rudderstack.sdk.flutter.integrations.appcenter_example;
+
+import io.flutter.embedding.android.FlutterActivity;
+
+public class MainActivity extends FlutterActivity {
```
Comment (arnab-p) [inline integrations/rudder-integration-appcenter-flutter/example/android/app/src/main/java/com/rudderstack/sdk/flutter/integrations/appcenter_example/MainActivity.java:5]: Let's discuss once

**Review by arnab-p — COMMENTED** [review]
Let's review the `.gitignore` files on the discussion

File: integrations/rudder-integration-appcenter-flutter/example/android/app/src/main/java/com/rudderstack/sdk/flutter/integrations/appcenter_example/MainActivity.java (hunk 3 above)
Comment (desusai7) [reply integrations/rudder-integration-appcenter-flutter/example/android/app/src/main/java/com/rudderstack/sdk/flutter/integrations/appcenter_example/MainActivity.java:5]: Yeah Sure

**Review by arnab-p — APPROVED** [review]

**Pull request merged**